Automattic/wp-calypso
# Check if user has unlimited subscriber feature

Fix https://github.com/Automattic/wp-calypso/issues/86103

## Proposed Changes

- Adds check to see if site has unlimited subscribers feature

## Testing Instructions

Code review should probably be fine.

- Test for regressions in the following areas:
   - Stepper flow: Start at /setup/newsletter
   - Add subscribers modal:  users > subscribers > "+ Add subscribers"
- Test with a plan that has the `unlimited-subscribers` feature

## Pre-merge Checklist
